pkg/services/provisioning/alerting/rules_provisioner.go

```diff
@@ -97,8 +96,9 @@ func (prov *defaultAlertRuleProvisioner) provisionRule(
 func (prov *defaultAlertRuleProvisioner) getOrCreateFolderUID(
 	ctx context.Context, folderName string, orgID int64) (string, error) {
 	cmd := &dashboards.GetDashboardQuery{
-		Slug:  slugify.Slugify(folderName),
-		OrgID: orgID,
+		Title:    &folderName,
+		FolderID: util.Pointer(int64(0)),
```

Comment:
that's can be fine for now but if nested folders are enabled then we need also the parent folder information